commands/delete-database.go

```diff
@@ -124,10 +124,13 @@ func deleteDatabase(c *cli.Context) error {
 	return nil
 }
 
-func deleteSingleDatabase(res *resources.Resources, dbnames []string, db string, dryRun bool) error {
+func deleteSingleDatabase(res *resources.Resources, db string, dryRun bool) error {
 	// check if database exists
-	dbExists := util.StringInSlice(db, dbnames)
```

Comment:
This was an unnecessary parameter. In each call site, the value passed as db was obtained by the iterating over the value passed as dbnames. I've replaced the ineffectual check with one that will check if there is any data in the given db.